### PR TITLE
Update hyperscale-architecture.md

### PR DESCRIPTION
**Pull request by KoprowskiT**
A few changes with grammar, punctuation, and wording.

### CODE REVIEW

File: azure-sql/database/hyperscale-architecture.md [hunk 1]
```diff
@@ -19,37 +19,37 @@ The [Hyperscale service tier](service-tier-hyperscale.md) utilizes an architectu
...
 
 All network communication among Hyperscale components uses Azure network infrastructure with built-in redundancy.
 
-High availability secondary replicas and named replicas are optional compute nodes which can be added on-demand. Both share the same storage components, so no data copy is required to spin up a new replica. A geo secondary replica can be added on-demand in same or different Azure region. For data protection and redundancy, geo secondary replicas have storage components that are separate from those used by primary replica.
+High-availability secondary replicas and named replicas are optional compute nodes that can be added on demand. Both share the same storage components, so no data copy is required to spin up a new replica. A geo-secondary replica can be added on-demand in the same or different Azure regions. For data protection and redundancy, geo-secondary replicas have storage components that are separate from those used by primary replicas.
```
Comment (dimitri-furman) [inline azure-sql/database/hyperscale-architecture.md:28]:
```suggestion
High-availability secondary replicas and named replicas are optional compute nodes that can be added on demand. Both share the same storage components, so no data copy is required to spin up a new replica. A geo-secondary replica can be added on demand in the same or a different Azure region. For data protection and redundancy, geo-secondary replicas have storage components that are separate from those used by the primary replica.
```

File: azure-sql/database/hyperscale-architecture.md [hunk 2]
```diff
@@ -19,37 +19,37 @@ The [Hyperscale service tier](service-tier-hyperscale.md) utilizes an architectu
...
+Hyperscale databases have one primary compute node where the read-write workload and transactions are processed. Up to four high-availability secondary compute nodes can be added on demand. They act as hot standby nodes for failover purposes and may serve as read-only compute nodes to offload read workloads when desired. [Named replicas](service-tier-hyperscale-replicas.md#named-replica) are secondary compute nodes designed to enable a variety of additional OLTP [read-scale out](read-scale-out.md) scenarios and to support better Hybrid Transactional and Analytical Processing (HTAP) workloads. A [geo secondary](active-geo-replication-overview.md) compute node can be added for disaster recovery purposes and to serve as a read-only compute node to offload read workloads in a different Azure region.
```
Comment (dimitri-furman) [inline azure-sql/database/hyperscale-architecture.md:42]: The original placement of "better" is correct here. The sentence refers to better support, not better workloads.

**Review by dimitri-furman — CHANGES_REQUESTED** [review]
Thanks for the edits to make the article clearer and more consistent! I added a couple of minor suggestions. Please commit them if you agree and we'll get this merged.

File: azure-sql/database/hyperscale-architecture.md (hunk 1 above)
Comment (rwestMSFT) [reply azure-sql/database/hyperscale-architecture.md:28]: @dimitri-furman I'm including these in the internal PR https://github.com/MicrosoftDocs/sql-docs-pr/pull/28663.

**Comment by rwestMSFT** [comment]
Replaced by internal PR https://github.com/MicrosoftDocs/sql-docs-pr/pull/28663.

@KoprowskiT We use the Oxford comma, so I removed a few of your edits. As my colleague also noted, Azure SQL Database is the name of the service, so other edits were redacted.

It will take about a day for the internal PR to be approved and published, at which point you will see the updated content.

Thank you again for your contributions. We appreciate your close attention to our content.

**Pull request closed without merging**